src/scores/pandas/__init__.py

```diff
@@ -0,0 +1,5 @@
+"""
+Explicit Pandas API
+"""
+
```

Comment:
If you look at scores.__init__.py, it imports the other namespaces rather than leaving it to the user to import each one. Importing things is slow in Python unfortunately. I ran some timing test and the additional time to import scores.pandas by default is small, maybe about 0.04 seconds. I think for consistency I'd suggest adding "import scores.pandas" to the root __init__.py, but I could be convinced otherwise.

Reply:
Are we wanting to always expose the pandas api?
Do we want:
```python
import scores
scores.pandas.----
```
or
```python
import scores.pandas as scores
scores.---
```
Depending on the usage we want to suggest, will change if we import `pandas` into the root `__init__`

Reply:
I think I prefer the first one. There are pros and cons. The second one is shorter for people who always use pandas. However, there is other code around beyond just metric implementations. Such as scores.utils, scores.processing etc. Importing scores.pandas as scores will then mean they're not in the API for the user any more unless we also re-implement or re-import them into the pandas API. We could call the interface scores.pd to make it a little shorter potentially. For people wanting things to be really short, they could import the individual function, e.g. from scores.pandas import mse. Let's hear from some others too...

Reply:
Okay, I shall update it for now